.vscode/launch.json

```
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Listen for Xdebug",
+      "type": "php",
+      "request": "launch",
+      "port": 9003,
+      "pathMappings": {
+        "/var/www/html/wp-content/plugins/remote-data-blocks": "${workspaceFolder}/"
+      }
+    }
```

```suggestion
    },
    {
      "type": "chrome",
      "request": "launch",
      "name": "Launch Chrome against localhost",
      "url": "http://localhost:8888/",
      "webRoot": "${workspaceFolder}"
    }
```

While we're at it, I use this one regularly to debug the frontend (I like to keep a Chrome profile per project).